Formula/immudb.rb

```diff
@@ -42,6 +49,10 @@ def install
     response = shell_output(command.join(" "))
     assert_equal "world", Base64.decode64(JSON.parse(response)["value"])
 
-    assert_match "Uptime", shell_output("#{bin}/immuadmin stats -t -p #{immudb_port}")
+    shell_output("#{bin}/immuadmin stats -t -p #{immudb_port}")
+
+    pid = File.read(pidfile)
+
+    Process.kill("KILL", pid.to_i)
```

Comment:
All test processes will be killed by brew after the test, no need to do it manually

Reply:
Thanks, changed.